src/wkt/tests/unknown.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Test serialization for unknown fields.
+//!
+//! This shows how the generator could handle unknown fields in the future.
+
+#[cfg(test)]
+mod test {
+    use serde_json::json;
+
+    #[test]
+    fn roundtrip_from_json() -> anyhow::Result<()> {
+        let input = json!({
+            "field0": "abc123",
```

Comment:
might also want a test for a known field which is a message type, and that message has an unknown field.

Reply:
Done.